app.js

```diff
@@ -421,6 +442,29 @@ app.post('/uploadimage', function (req, res) {
         }
     });
 });
+//get gitlab parameters
+app.get('/gitlab', function (req, res) {
+    var ret = { baseURL: config.gitlab.baseURL };
+    models.User.findById(req.cookies.userid)
```

Comment:
Please look at `/me` api, it already show how we handle to find a user, no need use cookie parser again.